### PR TITLE
Fix bug secrets escape

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
handling escape characters from whitelist.json.

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/secrets/tests/secrets_test.py [hunk 1]
```diff
@@ -172,9 +180,17 @@ def test_remove_white_list_regex(self):
...
+        file_contents = self.validator.remove_whitelisted_items_from_file(file_contents, white_list)
         assert white_list not in file_contents
 
+    def test_remove_whitelisted_items_from_file_escaped_whitelist(self):
+        white_list = '***.url'
+        file_contents = '''
+        Random and unmeaningful file content
+        '''
+        file_contents = self.validator.remove_whitelisted_items_from_file(file_contents, white_list)
+        assert file_contents
```
Comment (Itay4) [inline demisto_sdk/commands/secrets/tests/secrets_test.py:192]: What are we checking here?

Comment (jochman) [reply demisto_sdk/commands/secrets/tests/secrets_test.py:192]: The secret contains `***.url` which is an invalid regex that causes the system to raise an exception.

**Comment by Itay4** [comment]
And how exactly that test checks that?

**Comment by jochman** [comment]
> And how exactly that test checks that?

I've found that this term failed when running secrets. So I fixed it and added a unittest to see that it's not failing again.

**Comment by Itay4** [comment]
It is extremely unclear... let’s clarify what we’re testing here in the test docstring and let’s assert something that actually checks for something

**Pull request merged**